package/yast2-cio.changes

```diff
@@ -1,7 +1,15 @@
 -------------------------------------------------------------------
+Fri Jul 31 11:55:51 UTC 2015 - jreidinger@suse.com
+
+- fix building with new rspec
+- 3.1.8
+
+-------------------------------------------------------------------
 Thu Dec  4 07:13:51 UTC 2014 - jreidinger@suse.com
 
 - remove X-KDE-Library (bnc#899104)
+- 3.1.7
```

Comment:
I'm not sure here about this change, the OBS check might report non-top change...